### DISCUSSION TITLE
Jetbot with Jetpack

### DISCUSSION THREAD

**Discussion by m-hameds** [NVIDIA-AI-IOT/jetbot#593] [General]
Hi, can you help me with my issue?
I have installed jetpack on jetson nano nvidia-l4t-core 32.4.3-20200625213809. I
the jupyter container is run using /home/nvidia/docker_dli_run.sh.

I was trying to install jetbot on the same jetson nano. Using the commands

git clone github.com/NVIDIA-AI-IOT/jetbot.git
./jetbot/scripts/configure_jetson.sh
sudo systemctl set-default graphical.target
./scripts/re_enable_gui.sh
./scripts/enable_swap.sh
cd docker
./enable.sh $HOME

I got the following message
JETBOT_VERSION=0.4.3
L4T_VERSION=32.4.3
JETBOT_BASE_IMAGE=[nvcr.io/nvidia/l4t-pytorch:r32.4.3-pth1.6-py3](http://nvcr.io/nvidia/l4t-pytorch:r32.4.3-pth1.6-py3)
Restarting docker daemon …
Synchronizing state of docker.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable docker
Unable to find image ‘jetbot/jetbot:display-0.4.3-32.4.3’ locally
docker: Error response from daemon: manifest for jetbot/jetbot:display-0.4.3-32.4.3 not found: manifest unknown: manifest unknown.
See ‘docker run --help’.
Unable to find image ‘jetbot/jetbot:jupyter-0.4.3-32.4.3’ locally
docker: Error response from daemon: manifest for jetbot/jetbot:jupyter-0.4.3-32.4.3 not found: manifest unknown: manifest unknown.
See ‘docker run --help’.
nvidia@nvidia-desktop:~/jetbot/docker$ cat /etc/nv_tegra_release

R32 (release), REVISION: 4.3, GCID: 21589087, BOARD: t210ref, EABI: aarch64, DATE: Fri Jun 26 04:38:25 UTC 2020
nvidia@nvidia-desktop:~/jetbot/docker$ dpkg-query --show nvidia-jetpack
dpkg-query: no packages found matching nvidia-jetpack
nvidia@nvidia-desktop:~/jetbot/docker$ dpkg-query --show nvidia-jetpack
dpkg-query: no packages found matching nvidia-jetpack
nvidia@nvidia-desktop:~/jetbot/docker$ dpkg-query --show nvidia-l4t-core
nvidia-l4t-core 32.4.3-20200625213809

Are they incompatible? can you please advise?
If both containers are working, how to assign different IPs to each one?

Thank you so much for your help

**Comment by jaybdub** [comment]
Hi @m-hameds ,

Thanks for reaching out!

Apologies for the delayed response.

Unfortunately, we don't have a pre-built container in our registry for JetBot version (0.4.3) + L4t version (32.4.3).

The latest version of JetBot we have that targets L4T 32.4.3 is JetBot 0.4.1.

### Option 1 - Build containers from scratch

One solution may be to build the containers for JetBot 0.4.3 + L4T 32.4.3 from scratch.  To do so, call the following 

```bash
source configure.sh
./build.sh
```

And then you should be able to run the commands as you did before, because the images will be found locally.

### Option 2 - Revert JetBot version

Alternatively, you could revert the JetBot version to the last version matching L4T 32.4.3.
```bash
cd jetbot
git checkout v0.4.1
```
And then run the commands as before.

Please let me know if this helps or if you run into any issues.

Thanks!
John
